src/components/molecules/MyComponent/MyComponent.tsx

```diff
@@ -1,9 +1,17 @@
 import { CustomDatePicker } from "@/components/atoms/DatePicker/DatePicker";
 
 export function MyComponent() {
+  const endDateLimit = new Date();
+  endDateLimit.setDate(endDateLimit.getDate() + 30);
   return (
     <div className="w-full flex justify-center">
-      <CustomDatePicker isEndDate />
+      <CustomDatePicker isEndDate 
+                        startDatePlaceHolder="Start Date"
+                        endDatePlaceHolder="End Date"
+                        calendarStartFromMondayDay={false}
+                        endDateLimit={endDateLimit}
+                        markSundayAsRed={true}
```

Comment:
It just `markSundayAsRed`, it's read as `true` flag.

Reply:
updated;